src/components/VideoPlayer/VideoPlayer.tsx

```diff
@@ -54,29 +57,61 @@ export default function VideoPlayer(
       ? `https://player.vimeo.com/video/${videoId}?autoplay=0&loop=0`
       : `https://www.youtube.com/embed/${videoId}?disablekb=1&autoplay=0&fs=1&modestbranding=0`;
 
+  const iFramTitleEmbedCode = iframeTitle ? `${iframeTitle}` : `Video player`;
```

Comment:
Small typo, missing "e" in `iFramTitleEmbedCode`.

Reply:
Good catch.  Thought I had fixed that.